dev-tools/xtask/Cargo.toml

```diff
@@ -31,8 +31,13 @@ clap.workspace = true
 fs-err.workspace = true
 macaddr.workspace = true
 serde.workspace = true
+serde_json.workspace = true
```

Comment:
There was interest in keeping `xtask` as tiny as possible and using the "external" pattern (which is also pretty easy) to have it invoke other commands instead.

I don't have an opinion about whether this should be "external" or not.